spark/dl/src/main/resources/serialization/bigdl.proto

```diff
@@ -16,7 +16,7 @@ message BigDLModule
     string namePostfix = 11; // name post fix
     int32 id = 12; // unique ID of this module , used for shared modules
     Shape inputShape = 13; // input shape
-    repeated Shape outputShape = 14; //output shape
```

Comment:
both inputShape and outputShape should be optional (default to null) to ensure backward compatibility?

Reply:
We just remove "repeated" here and it would be backward compatible with the old model. cc @wzhongyuan

Reply:
@jason-dai   it's by default optional in proto3